src/formatters/stylishFormatter.ts

```diff
@@ -20,6 +20,7 @@ import {IFormatterMetadata} from "../language/formatter/formatter";
 import {RuleFailure} from "../language/rule/rule";
 
 import * as colors from "colors";
+const codeFrame = require("babel-code-frame");
```

Comment:
I think you'll need to disable the `no-var-requires` rule to get your build passing. This is fine for now, just leave a comment explaining that the typings for this are currently in progress.